beast/fitting/fit.py

```diff
@@ -262,7 +264,7 @@ def Q_all_memory(prev_result, obs, sedgrid, ast, qnames_in, p=[16., 50., 84.],
 
     # get the names of all the children in the ast structure
     ast_children = []
-    for label, node in ast.root._v_children.items(): 
+    for label, node in list(ast.root._v_children.items()): 
```

Comment:
Yes.  I've been trying to move to h5py.  But this move is not complete as you might imagine.  Part of the move to astropy tables.